tests/pytests/unit/beacons/test_smartos_vmadm.py

```diff
@@ -0,0 +1,240 @@
+# Python libs
+import pytest
+
+# Salt libs
+import salt.beacons.smartos_vmadm as vmadm
+from tests.support.mock import MagicMock, patch
+
+# Mock Results
+MOCK_CLEAN_STATE = {"first_run": True, "vms": []}
```

Comment:
we shouldn't be using mutable objects at the global level ( https://docs.saltproject.io/en/master/topics/development/tests/index.html#module-global-level-variables)

This includes this dictionary and the ones below.